# An array of goal messages for a robot to decide between
Goal[] goals
std_msgs/Float32 costHome
